dev/defs/assets/layer_image_asset.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "LayerImageAsset",
```

Comment:
how is this file related to the change(s)?

Reply:
I updated the core defs, it syncs all the definition json files and then ran the generator. There's no
way to piece-meal it as the json defs are meant to be an atomic set. It didn't get run while I was out so some other things got picked up.

Reply:
See dev/update_defs.sh for details on how it syncs. Basically just grabs subtree from rive. Wouldn't be a problem with everything in rive :)